package.json

```diff
@@ -34,7 +34,7 @@
     "@types/express": "4.17.17",
     "@types/jest": "^26.0.19",
     "@types/nconf": "^0.10.0",
-    "@types/node": "^14.14.20",
+    "@types/node": "^20.5.7",
```

Comment:
I think we should go with the latest for the minimum version of node we support.

Reply:
Fair. We should really upgrade the version though - I believe prod is still on v14.